Toolbox/scripts/dump_pf.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+SCRIPTDIR=$( cd -P -- "$(dirname -- "$(command -v -- "$_")")" && pwd -P )
+
+#info
+TOPIC=Firewall
+DESCRIPTION="This script will dump the pf (firewall) config."
+
+#Volumes/files
+FILENAME=pf.conf
+ORIGINAL=/net/mmx/mnt/system/etc/
+
+echo $DESCRIPTION
+
+. ${SCRIPTDIR}/util_info.sh
+. ${SCRIPTDIR}/util_mountsd.sh
+if [[ -z "$VOLUME" ]] 
+then
+	echo "No SD-card found, quitting"
+	exit 0
+fi
+
+#Make dump folder
+DUMPFOLDER=$VOLUME/Dump/$VERSION/$FAZIT/$TOPIC
+
+echo Dump-folder: $DUMPFOLDER
+mkdir -p $DUMPFOLDER
+echo Dumping, please wait. This can take a while.
+sleep 1
+
+echo Copying ${FILENAME} file to SD-card.
+cp $ORIGINAL/$FILENAME $DUMPFOLDER/${FILENAME}
```

Comment:
What is the difference here in Filename with {} and without? Why are we doning this?

Reply:
Ah, in this case there shouldn't be a difference. Adding the `{}` is a good habit I got into, it makes it clearer for the shell exactly what is the variable you're referencing. It allows you to do things like `NAME="FOO"; MYPATH=${NAME}BAR` which will mean MYPATH = FOOBAR. If you instead did `MYPATH=$NAMEBAR` it'll search for the wrong variable.